everware/tests/test_repourl_parser.py

```diff
@@ -0,0 +1,40 @@
+from .. import git_executor
+
+def test_parser():
+    tests = [
+        (
+            'git://github.com/aladagemre/django-notification.git@2927346f4c513a217ac8ad076e494dd1adbf70e1',
```

Comment:
should we translate git:// into https:// prefix on the server side since git requires ssh key to check permission and will fail for sure?